src/GigaSOM.jl

```diff
@@ -18,7 +18,7 @@ module GigaSOM
     # using Statistics
     # using StatsBase
     # using StatsPlots
-    # using DistributedArrays
+    using DistributedArrays
     # using FCSFiles
     # using JuliaInterpreter
     # using LinearAlgebra
```

Comment:
`ERROR: LoadError: could not open file /mnt/artenolis-data/workspace/GigaSOM.jl-pr-auto-linux/JULIA_VER/v1.0.3/label/GigaSOM/src/batch_som.jl`

May you remove `include("batch_som.jl")`?